src/pages/settings/Subscription/CardSection/CardSection.tsx

```diff
@@ -89,6 +122,16 @@ function CardSection() {
                 )}
                 {isEmptyObject(defaultCard?.accountData) && <CardSectionDataEmpty />}
             </View>
+            {billingStatus?.isRetryAvailable !== undefined && (
```

Comment:
So we're saying if it's false the button will be shown disabled?

Reply:
Yes. We have various scenarios here:
- do not display the button if `isRetryAvailable` is not returned at all from the billing status
- display it but make it disabled (`isRetryAvailable` is equal to false)
- display it as enabled button (`isRetryAvailable` is equal to true)

Im not sure if you have access to design doc but the cases are shown and described under `Payment failed` subsection